cli/migrate.go

```diff
@@ -11,40 +11,50 @@ import (
 	_ "github.com/mattes/migrate/source/file"
 )
 
+// MigrationLogger abstraction for migrate-logger
 type MigrationLogger struct {
 	logger logger.Logger
 }
 
+// NewMigrationLogger creates abstraction for migrate-logger
 func NewMigrationLogger(logger logger.Logger) MigrationLogger {
 	return MigrationLogger{
 		logger: logger,
 	}
 }
 
+// Printf implementation of migrate.Logger
```

Comment:
`Printf implementation of migrate.Logger` -> `Printf implementation of migrate.Logger.Info` ?